topo/node/cptx/cptx.go

```diff
@@ -5,16 +5,28 @@ package cptx
 
 import (
 	"context"
+	"errors"
 	"fmt"
 	"io"
+	"io/ioutil"
+	"time"
 
 	topopb "github.com/google/kne/proto/topo"
 	"github.com/google/kne/topo/node"
+	scraplibase "github.com/scrapli/scrapligo/driver/base"
+	scraplicore "github.com/scrapli/scrapligo/driver/core"
+	scraplinetwork "github.com/scrapli/scrapligo/driver/network"
+	scraplitransport "github.com/scrapli/scrapligo/transport"
+	scraplicfg "github.com/scrapli/scrapligo/cfg"
+	log "github.com/sirupsen/logrus"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 	"google.golang.org/protobuf/proto"
 )
 
+// ErrIncompatibleCliConn raised when an invalid scrapligo cli transport type is found.
+var ErrIncompatibleCliConn = errors.New("incompatible cli connection in use")
```

Comment:
Why does this need to be exported ?

Reply:
Just a new custom explicit value of error when an invalid scrapligo cli transport type is found. We need to patch the cli invocation commands as we have to invoke cli executable instead of logging into the device. Do let me know if we can do this better. This pretty much comes from the latest example in ceos.go.